core/src/display_object/container.rs

```diff
@@ -289,6 +289,10 @@ pub trait TDisplayObjectContainer<'gc>:
     }
 
     /// Iterates over the children of this display object in depth order.
+    /// Unless you are performing an AVM1-specific operation, you should
```

Comment:
The only remaining usage of `iter_depth_list` is the Ctrl+Alt+V AVM1 `VariableDumper` in `player.rs`. I think it's a good opportunity to greatly simplify `iter_depth_list` (along with `DepthIter` and `lowest_depth`), maybe by iterating over an `i32` and call `child_by_depth` each time?